### ISSUE TITLE
Notification Top Center Position Bug

### ISSUE THREAD

**Issue by bribar** [uikit/uikit#4208]
If a notification pos is set to top-center it does not show, or is added to the DOM like the other pos options. The five others work as expected. I have reverted back to 3.3.2 and top-center works as expected, so I believe it has something with 3.3.3 and beyond.

Something to do with this? "Notifications remove their container if it's empty", but it's not getting added to DOM for this to take place.

I use UIkit with Vuejs but I don't think that's a difference because other positions work. I don't override css in my project.

### UIkit Version

3.3.3 - 3.4.6

How I am using the code:

    UIkit.notification({
			message: 'Message here!',
			status: 'primary',
			pos: 'top-center',
			timeout: 5000
		});

**Comment by janschoenherr** [comment]
This seems to work fine for me. Please add a reproduction link.

**Comment by bribar** [comment]
@janschoenherr 
 
I guess it's not a bug. I realized after further testing that I implemented my own notification with the same structure 

    <div class="uk-notification uk-notification-top-center">

to get the same look and feel, but have custom control that my situation required. I guess with updating to more recent versions, UIkit doesn't like that a notification with uk-notification-top-center already exists in the DOM. I was on pre-version3.3.3 and this worked fine.

Sorry about that, it threw me off that other positions worked, but top-center didn't.
